.github/workflows/test.yml

```diff
@@ -35,17 +35,21 @@ jobs:
     - run: npm test
   tox:
     name: >-
-      ${{ matrix.env.TOXENV }}
+      ${{ matrix.tox_env }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version:
-        - 3.8
-        os:
-        - ubuntu-20.04
-        env:
-        - TOXENV: py,lint
+        include:
+        - python-version: 3.8
+          os: ubuntu-20.04
+          tox_env: py38,lint
```

Comment:
Does lint need to be run under every Python runtime possible?

Reply:
running lint on the latest should be sufficient. I'll change it.

Reply:
Maybe put it into a separate job?

Reply:
done as suggested